Assignment_1/palindrome_permutation.js

```diff
@@ -0,0 +1,33 @@
+const palindromePermutation = (str) => {
+
+    const charMap = new Map(); //frequency of each char in string
+    str = str.replace(/\s/g, '').toLowerCase();
```

Comment:
Good use of regular expressions here. However, it's typically not a good practice to modify the incoming arguments when possible. In this case, it would be better to create a new variable instead of reassigning to `str`.